terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts

```diff
@@ -70,27 +70,41 @@ export function resetRunnersCaches() {
   ssmParametersCache.reset();
 }
 
+// Helper function to get the current AWS account ID
+async function getCurrentAccountId(): Promise<string> {
+  const sts = new STS();
+  const identity = await sts.getCallerIdentity().promise();
```

Comment:
Please double check to see if the lambdas have access to this identity before checking in the code. But I suspect my comment below includes all the identities that would be needed